lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -219,6 +219,20 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._tagNameControl;
     }
 
+    /*
+     * Returns `true` if tagNameControlVisible is true, otherwise `false`
+    */
+    get isNameTagsVisible(): boolean {
+        return (!this.tagNameControlVisible && this.tags?.length > 0) || this.tagNameControlVisible;
+    }
+
+    /*
+     * Returns `true` if tags empty, otherwise `false`
+    */
+    get isTagsEmpty(): boolean {
+        return this.tags?.length === 0 && !this.tagNameControlVisible;
```

Comment:
Same here, why do you check tag name control visibility inside a method that should return the info if tags are empty?

Reply:
This condition is needed if we remove this condition it shows in view mode and edit mode too. As per the design we have to show the empty message in view mode only.

Reply:
If you need both conditions then the name of the method is confusing and should be adjusted

Reply:
Updated the appropriate method name